src/assets/i18n/en.json5

```diff
@@ -4029,6 +4029,8 @@
 
   "mydspace.new-submission": "New submission",
 
+  "mydspace.new-submission.not-available": "You can only have one pending submission at the time.",
```

Comment:
"A dissertation has already been submitted and is still in validation workflow. Please contact your faculty manager to resolve any problem."

Reply:
'disseration' ou 'thesis' ??